### PR TITLE
Removed versions for UsingRegistry

### PR DESCRIPTION
**Pull request by martinvol**
### Description

Removed getversions so the tooling doesn't try to deploy it.

### CODE REVIEW

File: packages/protocol/contracts/common/UsingRegistryV2.sol [hunk 1]
```diff
@@ -79,14 +79,6 @@ contract UsingRegistryV2 {
     _;
   }
 
-  /**
-   * @notice Returns the storage, major, minor, and patch version of the contract.
-   * @return The storage, major, minor, and patch version of the contract.
-   */
-  function getVersionNumber() external pure returns (uint256, uint256, uint256, uint256) {
```
Comment (eelanagaraj) [inline packages/protocol/contracts/common/UsingRegistryV2.sol:86]: any reason not to remove this from `UsingRegistryV2BackwardsCompatible` as well?

File: packages/protocol/scripts/bash/contract-exclusion-regex.sh [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+
+CONTRACT_EXCLUSION_REGEX=".*Test|Mock.*|I[A-Z].*|.*Proxy|MultiSig.*|ReleaseGold|SlasherUtil|UsingPrecompiles"
+
+# Before CR7, they had been deployed, so don't rebuild the report
+VERSION_NUMBER=$(echo "$OLD_BRANCH" | tr -dc '0-9')
+
+if [ $VERSION_NUMBER -gt 6 ]
+  then
+  CONTRACT_EXCLUSION_REGEX="$CONTRACT_EXCLUSION_REGEX|^UsingRegistry"
```
Comment (eelanagaraj) [inline packages/protocol/scripts/bash/contract-exclusion-regex.sh:10]: are there any risks in excluding UsingRegistry from comparisons? or is this absolutely required?

**Review by eelanagaraj — COMMENTED** [review]
just have a couple questions but can approve right after!

File: packages/protocol/contracts/common/UsingRegistryV2.sol (hunk 1 above)
Comment (martinvol) [reply packages/protocol/contracts/common/UsingRegistryV2.sol:86]: Done, forgot to check in the file

File: packages/protocol/scripts/bash/contract-exclusion-regex.sh (hunk 2 above)
Comment (martinvol) [reply packages/protocol/scripts/bash/contract-exclusion-regex.sh:10]: UsingRegistry shouldn't have been deployed in the first place either, as they were supposed to be mixin. Deploying it looks like a bug in the tooling all along.

File: packages/protocol/scripts/bash/contract-exclusion-regex.sh [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Exclude test contracts, mock contracts, contract interfaces, Proxy contracts, inlined libraries,
+# MultiSig contracts, and the ReleaseGold contract.
+
+
+
```
Comment (martinvol) [inline packages/protocol/scripts/bash/contract-exclusion-regex.sh]:
```suggestion

```

File: packages/protocol/scripts/bash/contract-exclusion-regex.sh [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+CONTRACT_EXCLUSION_REGEX=".*Test|Mock.*|I[A-Z].*|.*Proxy|MultiSig.*|ReleaseGold|SlasherUtil|UsingPrecompiles"
+
+# Before CR7, they had been deployed, so don't rebuild the report
```
Comment (martinvol) [inline packages/protocol/scripts/bash/contract-exclusion-regex.sh]:
```suggestion
# Before CR7, UsingRegistry and UsingRegistryV2 had been deployed, they need to keep getting deployed to keep the release reports without changes.
```

**Review by eelanagaraj — APPROVED** [review]
looks good!

**Pull request merged**